### PR TITLE
drafting intro for docs.md

### PR DESCRIPTION
**Pull request by kanitw**

### CODE REVIEW

File: site/docs/docs.md [hunk 1]
```diff
@@ -5,9 +5,14 @@ title: Overview
 permalink: /docs/
 ---
 
-This documentation describes [Vega-Lite specifications](#spec) and the functionalities of the [Vega-Lite library](#lib).
+**Vega-Lite** is a high-level visualization grammar.  It provides a concise JSON syntax for supporting rapid generation of visualizations to support analysis.  Vega-Lite can serve as a declarative format for describing and creating data visualizations.  Vega-Lite specifications can be compiled to a lower-level, more detailed specifications in the form of [Vega](http://vega.github.io/vega) specifications and rendered using Vega's compiler.
```
Comment (domoritz) [inline site/docs/docs.md]: I don't think it's good that the docs repeat what we already write on the homepage. Do we want to keep the home page shorter then?

File: site/docs/docs.md [hunk 2]
```diff
@@ -5,9 +5,14 @@ title: Overview
...
+
+At its core, Vega-Lite describes visualizations as mappings between data to properties of graphical marks (e.g., points or bars).  By simply providing a mark type and a mapping, Vega-Lite automatically produces other visualization components including axes, legend, and scale, which transforms data values from the data domain to the visual output domain.  Unless explicitly specified, Vega-Lite determines properties of these components based on a set of carefully designed rules.  This approach allows Vega-Lite specifications to be succinct and expressive but also provides user-control.  
+
+As it is design for analysis, Vega-Lite also supports data transformation such as [aggregation](aggregate.html), [binning](bin.html), [time unit conversion](timeunit.html), [filtering](transform.html), and [sorting](sort.html).  In addition, it also supports faceting a single plot into [trellis plots or small multiples](https://en.wikipedia.org/wiki/Small_multiple).
 
 {:#spec}
+
```
Comment (domoritz) [inline site/docs/docs.md]: why the empty line?

File: site/docs/docs.md (hunk 1 above)
Comment (kanitw) [inline site/docs/docs.md]: Or we can move this paragraph to the home page.

Comment (domoritz) [inline site/docs/docs.md]: Let's do that. Will merge for now.

**Pull request merged**